examples/prism-tsx.html

```diff
@@ -0,0 +1,34 @@
+<h1>React TSX</h1>
+<p>To use this language, use the class "language-tsx".</p>
+
+<h2>Full example</h2>
+<pre><code>import * as React from 'react';
+
+interface IState {
+	clicks: number;
+}
+
+export class Clicker extends React.Component<any, IState> {
```

Comment:
I think you must escape all `<` characters in this file using `&lt;`.

You can ensure the example displays properly by updating your branch, and then changing [this line](https://github.com/PrismJS/prism/blob/gh-pages/examples.js#L9) to test your [Examples](http://prismjs.com/examples.html) page locally.